app/javascript/src/translations/fr.json

```diff
@@ -0,0 +1,3 @@
+{
+  "greeting": "France"
+}
```

Comment:
I don't understand the point of this translation file. Is this file needed?

Reply:
Nope, this file is not needed, just that I copied the translations dir from another repo which had this file, will remove it.